### PR TITLE
Time review

### PR DESCRIPTION
**Pull request by mgrippo**
Created an analyzer to calculate the number of event per job for prod_v3, starting from prod_v2 information

### CODE REVIEW

File: Instruments/source/TimeAnalyzer_t.cxx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+#include "AnalysisTools/Core/include/NumericPrimitives.h"
+
+
+struct Arguments {
+    run::Argument<std::string> input{"input", "Input file."};
+};
+
+namespace analysis {
+
+class TimeAnalyzer_t {
```
Comment (kandrosov) [inline Instruments/source/TimeAnalyzer_t.cxx]: Rename to ExeTimeAnalyzer (remove _t)

File: Instruments/source/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# How to run
+### Run merge analyzer for DYjets
+./run.sh NJets_HT_BinFileMerger summary /Users/Tita/Desktop hh-bbtautau/Instruments/config/dyjets_splitting.cfg hh-bbtautau/Instruments/config/DYJets_file.cfg hh-bbtautau/Analysis/config/dyjets_weights.cfg
```
Comment (kandrosov) [inline Instruments/source/README.md]: Use code sections for commands

File: Instruments/source/TimeAnalyzer_t.cxx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+        auto n_process_event = std::make_shared<TH1F>("n_process_event","number processed events",50000,-0.5,50000-0.5);
+        auto exeTime = std::make_shared<TH1F>("exeTime","exeTime",10000,-0.5,100000-0.5);
+        auto time_per_event = std::make_shared<TH1F>("time_per_event","time_per_event",10000,-0.5,100000-0.5);
+
+        for(const auto& summary : *summaryTuple) {            
+            n_process_event->Fill(summary.numberOfProcessedEvents);
+            exeTime->Fill(summary.exeTime);
+            if(summary.numberOfProcessedEvents == 0) continue;
+            double time_event = summary.exeTime/summary.numberOfProcessedEvents;
```
Comment (kandrosov) [inline Instruments/source/TimeAnalyzer_t.cxx]: double time_event = double(summary.exeTime)/summary.numberOfProcessedEvents;

File: Instruments/source/TimeAnalyzer_t.cxx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+            }
+        }
+
+        static constexpr int crab_dead_time = 12 * 60 * 60; // s
+        static constexpr char sep = ','; // ,
+
+        double scale_factor = crab_dead_time/common_exe_time;
+        size_t n_evt_per_job_prod_v2 = static_cast<size_t>(std::ceil(n_process_event->GetMean()));
+        size_t n_evt_per_job_prod_v3 = static_cast<size_t>(std::ceil(scale_factor * n_evt_per_job_prod_v2));
+        double time_event_calc = static_cast<double>(std::ceil(time_per_event->GetMean()));
```
Comment (kandrosov) [inline Instruments/source/TimeAnalyzer_t.cxx]: static_cast<double> seems to be redundant.

File: Instruments/source/TimeAnalyzer_t.cxx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    run::Argument<std::string> input{"input", "Input file."};
+};
+
+namespace analysis {
+
+class TimeAnalyzer_t {
+public:
+    TimeAnalyzer_t(const Arguments& _args) : args(_args)
+    {
+//        std::cout << "inputFile: " << args.input() << std::endl;
```
Comment (kandrosov) [inline Instruments/source/TimeAnalyzer_t.cxx]: remove comment.

File: Instruments/source/README.md (hunk 2 above)
Comment (mgrippo) [inline Instruments/source/README.md]: Done

File: Instruments/source/TimeAnalyzer_t.cxx (hunk 1 above)
Comment (mgrippo) [inline Instruments/source/TimeAnalyzer_t.cxx]: Done

File: Instruments/source/TimeAnalyzer_t.cxx (hunk 5 above)
Comment (mgrippo) [inline Instruments/source/TimeAnalyzer_t.cxx]: Done

File: Instruments/source/TimeAnalyzer_t.cxx (hunk 3 above)
Comment (mgrippo) [inline Instruments/source/TimeAnalyzer_t.cxx]: Done

File: Instruments/source/TimeAnalyzer_t.cxx (hunk 4 above)
Comment (mgrippo) [inline Instruments/source/TimeAnalyzer_t.cxx]: Done

**Pull request merged**